extensions/jupyter-adapter/src/LanguageRuntimeAdapter.ts

```diff
@@ -708,6 +712,24 @@ export class LanguageRuntimeSessionAdapter
 		} as positron.LanguageRuntimeCommClosed);
 	}
 
+	/**
+	 * Converts a Jupyter clear_output message to a LanguageRuntimeMessage and
+	 * emits it.
+	 *
+	 * @param message The message packet
+	 * @param data The clear_output message
+	 */
+	onClearOutput(message: JupyterMessagePacket, data: JupyterClearOutput) {
+		this._messages.fire({
+			id: message.msgId,
+			parent_id: message.originId,
+			when: message.when,
+			type: positron.LanguageRuntimeMessageType.ClearOutput,
+			wait: data.wait,
+			metadata: message.metadata,
+		} as positron.LanguageRuntimeClearOutput);
```

Comment:
Ah, I wondered what the correct way is. I'll leave it as is for now since the same pattern is repeated a bunch in that file, but will consider `satisfies` moving forward.